oracle/src/confirmRelay.js

```diff
@@ -143,7 +143,12 @@ async function sendJobTx(jobs) {
   let nonce = await getNonce(web3Instance, ORACLE_VALIDATOR_ADDRESS)
 
   await syncForEach(jobs, async job => {
-    const gasLimit = addExtraGas(job.gasEstimate, EXTRA_GAS_PERCENTAGE)
+    let gasLimit
+    if (typeof job.extraGas === 'number') {
+      gasLimit = job.gasEstimate + job.extraGas
```

Comment:
Should we make sure that the calculated `gasLimit` is not bigger than `MAX_GAS_LIMIT` as it is done in the method `addExtraGas`?

Reply:
It's not likely to happen, but for consistency, I think it makes sense to do so. Addressed in 9f10bc9. Thanks!